container/compliance/tests/test_scan_codecs.py

```
@@ -0,0 +1,109 @@
+    p.write_bytes(b"\x7fELF")
+
+
+def test_in_tree_ffmpeg_is_allowed(tmp_path: Path):
+    # Our own /usr/local ffmpeg libs — must classify as allowed, never violate.
+    for rel in (
+        "usr/local/lib/libavcodec.so.62",
+        "usr/local/lib/libswscale.so.9",
+        "usr/local/bin/ffmpeg",
+    ):
+        _touch(tmp_path, rel)
+    violations, exceptions, allowed = scan_filesystem(tmp_path, _POLICY)
```

Ruff (0.15.21)

[warning] Unpacked variable `exceptions` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)